README.md

```diff
@@ -113,3 +113,13 @@ make -C ../jenkins.io run
 ```
 
 You can then browse to [Pipeline Steps Reference](http://localhost:4242/doc/pipeline/steps/) to see the running instance of `jenkins.io` on your `localhost`.
+
+### 6. Configure Parameters
+
+The parameters that need to be separated to new pages can be entered in `config.txt` by adhering to the following rules.
```

Comment:
You could also add another sentence about why you're making this change.  Just saying "this prevents having really long page loads" would be really helpful.